tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -142,7 +150,10 @@ async def test_train_persist_load_with_different_settings_non_windows(
         {"name": "DIETClassifier", MASKED_LM: True, EPOCHS: 1},
     ]
     await _train_persist_load_with_different_settings(
-        pipeline, component_builder, tmpdir
+        pipeline, component_builder, tmpdir, False
+    )
+    await _train_persist_load_with_different_settings(
```

Comment:
optional: consider using kwargs . Would make it easier to read in my opinion. E.g. just `False` doesn't tell much how this used with that many arguments.